packages/framework/src/Model/Product/Product.php

```diff
@@ -893,6 +905,109 @@ protected function setTranslations(ProductData $productData)
         }
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\ProductData $productData
+     */
+    private function setDomains($productData)
```

Comment:
Facades should have protected methods as they are supposed to be extended by inheritance.

Reply:
Also, the parameter should be type-hinted.

Reply:
It is not facade, but i think that this rule applies to entities too. Fixed